docker-app/qfieldcloud/filestorage/management/commands/migratefilestorage.py

```diff
@@ -47,8 +61,21 @@ def handle(self, *args, **options) -> None:
             assert owner_username_startswith is None
             assert owner_username_matches is None
 
+            if updated_until:
+                dt_nozoned = datetime.strptime(updated_until, "%Y-%m-%d")
```

Comment:
```suggestion
                dt_naive = datetime.fromisoformat(updated_until)
```

This should be fine, no? Also the term they use in Python is `naive`.

Reply:
Thanks !